packages/server/tracer.js

```diff
@@ -1,3 +1,14 @@
 import tracer from 'dd-trace'
-tracer.init() // initialized in a different file to avoid hoisting.
+tracer.init({
+  enabled: process.env.DD_TRACE_ENABLED === 'true',
+  env: process.env.DD_ENV ?? 'test'
+})
+
+tracer.use('graphql', {
```

Comment:
can we get rid of this `trace.use`? graphql doesn't exist on the webserver, so it should be safe to remove